development/playbooks/start-vms/start-vms.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Start VMs for development and testing
+  gather_facts: false
+  hosts:
+    - localhost
+  tasks:
+    - name: Run vagrant up
+      ansible.builtin.command:
+        cmd: "vagrant up {{ vms | default() }}"
+      args:
+        chdir: "{{ inventory_dir }}/../"
```

Comment:
This relative pathing feels kinda ugly but I couldn't find a good variable to use.